docs/source/prereqs.rst

```diff
@@ -1,10 +1,9 @@
 Prerequisites
 =============
 
-Before we begin, if you haven't already done so, you may wish to check that
-you have all the prerequisites below installed on the platform(s)
-on which you'll be developing blockchain applications and/or operating
-Hyperledger Fabric.
+Before you begin, you should confirm that you have installed all the prerequisites below on the platform where you will be running Hyperledger Fabric.
+
+.. note:: These prerequisites are recommended for Fabric users. If you are a Fabric developer, or want to contribute non-documentation content to Fabric, you should refer to the instructions for :doc:`dev-setup/devenv`.
```

Comment:
"or want to contribute non-documentation content to Fabric" seems awkward.
I think we can just remove.

Reply:
Done